presto-verifier/src/main/java/com/facebook/presto/verifier/checksum/ChecksumValidator.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.verifier.checksum;
+
+import com.facebook.presto.sql.tree.FunctionCall;
+import com.facebook.presto.sql.tree.QualifiedName;
+import com.facebook.presto.sql.tree.Query;
+import com.facebook.presto.sql.tree.Select;
+import com.facebook.presto.sql.tree.SelectItem;
+import com.facebook.presto.sql.tree.SingleColumn;
+import com.facebook.presto.sql.tree.Table;
+import com.facebook.presto.verifier.framework.Column;
+import com.facebook.presto.verifier.framework.Column.Category;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+
+import javax.inject.Inject;
+
+import java.util.List;
+import java.util.Map;
+
+import static com.facebook.presto.sql.QueryUtil.simpleQuery;
+import static com.facebook.presto.verifier.framework.Column.Category.FLOATING_POINT;
+import static com.facebook.presto.verifier.framework.Column.Category.ORDERABLE_ARRAY;
+import static com.facebook.presto.verifier.framework.Column.Category.SIMPLE;
+import static com.google.common.collect.ImmutableMap.toImmutableMap;
+
+public class ChecksumValidator
```

Comment:
Going with my comment above - how about `OutputValidator`

Reply:
Same as above: https://github.com/prestodb/presto/pull/12489#issuecomment-475096230

Reply:
I agree with @mayankgarg1990 here that since there's a checksum function, which is heavily used in the validation query, it's a bit misleading. The `ChecksumValidator` here obviously include non-checksum-function validations. I'm actually ok with just calling this `Validator`. `Output` seems to be redundant.

Reply:
There are multiple occurrences of the word `checksum` both in code and in general. For example, we always say `checksumQuery`, and everyone know what that means, but if we use `validationQuery`, it is not clear whether it is referring to the query that produces the comparison datum, or to the query being validated itself.